### PR TITLE
Feature/uttmm 61 auto set dateTime for baseEntity inheritors

### PR DESCRIPTION
**Pull request by RishkoPavel**

### CODE REVIEW

File: TheraLang.DLL/IttmmDbContext.cs [hunk 1]
```diff
@@ -1,15 +1,15 @@
 using Microsoft.EntityFrameworkCore;
-using TheraLang.DLL.Piranha.Configuration;
-using TheraLang.DLL.Piranha.Entities;
 using TheraLang.DLL.Configuration;
 using TheraLang.DLL.Entities;
+using TheraLang.DLL.Piranha.Configuration;
+using TheraLang.DLL.Piranha.Entities;
```
Comment (dgzargo) [inline TheraLang.DLL/IttmmDbContext.cs]: really helpful

File: TheraLang.DLL/UnitOfWork/UnitOfWork.cs [hunk 2]
```diff
@@ -20,6 +23,21 @@ public UnitOfWork(DbContext context)
...
+
+            foreach (var entityEntry in entries)
+            {
+                ((BaseEntity)entityEntry.Entity).UpdatedDateUtc = DateTime.Now;
+
+                if (entityEntry.State == EntityState.Added)
+                {
+                    ((BaseEntity)entityEntry.Entity).CreatedDateUtc = DateTime.Now;
+                }
+            }
```
Comment (dgzargo) [inline TheraLang.DLL/UnitOfWork/UnitOfWork.cs]: override it inside DbContext

File: TheraLang.DLL/IttmmDbContext.cs (hunk 1 above)
Comment (RishkoPavel) [reply TheraLang.DLL/IttmmDbContext.cs]: > really helpful

🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦
**why do you write write such comments?**

Comment (RishkoPavel) [reply TheraLang.DLL/IttmmDbContext.cs]: it's just a consequence of a program that removes and sorts unnecessary usages automatically

**Pull request closed without merging**